spec/support/common_feature_steps.rb

```diff
@@ -28,4 +28,31 @@ def given_I_am_a_2i_reviewer
   def required_permissions_for_2i
     ["signin", "GDS Editor", "2i reviewer", "Unreleased feature"]
   end
+
+  def and_I_am_the_step_by_step_author
+    login_as_user @review_requester
+  end
+
+  def and_I_am_the_reviewer
+    login_as_user @reviewer
+  end
```

Comment:
Should these be in step_nav_steps given that is where those variables are defined? I was a bit baffled where the variable came from.